bangazon-clumsy-penguins/BangazonSite
# Ajw/SD is local

# Added local delivery option 

## Description
Added checkbox on create product page to include local delivery option. If checkbox is checked then an input for city is shown.

##### Provide clarity to any potentially complex code:
Added client-side javascript to show/hide the city input depending on checkbox state

## Steps to Test
Delete current database and kill all connections.
Remove Migrations folder
add-migration
update-database
Login and click sell, make sure checkbox shows and hides city input field. Create item with city and without city.

##### Link to Feature Ticket
#39
